utilities/pipelines/platform/Set-AvmGitHubPrLabels.ps1

```diff
@@ -46,19 +46,21 @@ function Set-AvmGitHubPrLabels {
         # core team is already assigned, no or more than one module reviewer team is assigned
         if ($allTeamNames.Contains('avm-core-team-technical-bicep') -or $teamNames.Count -eq 0 -or $teamNames.Count -gt 1) {
             gh pr edit $pr.url --add-label 'Needs: Core Team :genie:' --repo $Repo
-        }
-        else {
+        } else {
             $teamMembers = [array](Get-GithubTeamMembersLogin -OrgName $Repo.Split('/')[0] -TeamName $teamNames[0])
 
             # no members in module team or only one and that user submitted the PR
             if (($teamMembers.Count -eq 0) -or ($teamMembers.Count -eq 1 -and $teamMembers[0] -eq $pr.author.login)) {
                 gh pr edit $pr.url --add-label 'Needs: Core Team :genie:' --repo $Repo
-            }
-            else {
+            } else {
                 gh pr edit $pr.url --add-label 'Needs: Module Owner :mega:' --repo $Repo
+                # add team members as reviewers
+                $teamMembers | ForEach-Object {
+                    gh pr review $pr.url --add-reviewer $_ --repo $Repo
+                }
```

Comment:
Just for my understanding, with this being merged
- The owner team is assigned to PRs as a reviewer (which is the status quo), though I somewhat don't see it in this code 😄 
- And now, in addition, each individual member of the owner team is also assigned as a reviewer
Is that accurate?

Reply:
@AlexanderSehr yes, as requested by @matebarabas